src/scripts/main.js

```diff
@@ -20,8 +20,32 @@ const food = {
 
 const tree = document.querySelector('#tree');
 
-function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+function createTree(element = document.createElement('ul'), data) {
```

Comment:
The default parameter for `element` is not used correctly. The default value `document.createElement('ul')` will never be used because `element` is always provided when `createTree` is called. Consider removing the default value or adjusting the function call.